shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/survey/model/SurveySubmissionParams.kt

```diff
@@ -0,0 +1,46 @@
+package co.nimblehq.mark.kmmic.data.service.survey.model
+
+import co.nimblehq.mark.kmmic.domain.model.SurveySubmission
+import co.nimblehq.mark.kmmic.domain.model.SurveySubmissionAnswer
+import co.nimblehq.mark.kmmic.domain.model.SurveySubmissionQuestion
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal data class SurveySubmissionParams(
```

Comment:
Is there any change with and without `internal`?

Reply:
We won't access this class from the native side, I aim to public the use case and models only 🙏

Reply:
Can you double check, on the iOS side, we can access those internal classes 🤔

Reply:
@suho 
![CleanShot 2023-05-10 at 14 49 03](https://github.com/markgravity/kmm-ic/assets/17875522/e771709b-676c-4ab1-a7bb-9e13b3cdd4dd)

Reply:
Cool! Thanks for the checking 🙌